Makefile

```diff
@@ -18,6 +20,9 @@ binary: clean
 	GOOS=linux GOARCH=amd64 govvv build -v \
 	  -ldflags "-X main.Version=`grep -E -m 1 -o  '<Version>(.*)</Version>' misc/manifest.xml | awk -F">" '{print $$2}' | awk -F"<" '{print $$1}'`" \
 	  -o $(BINDIR)/$(BIN) ./main 
+	GOOS=linux GOARCH=arm64 govvv build -v \
```

Comment:
For my information, does this build on an amd64 machine?

Reply:
Replacing with: uname -p